guides/real_time/presence.md

```diff
@@ -122,6 +124,120 @@ Bob (count: 1)
 
 If we close one of the Alice tabs, then the count should decrease to 1. If we close another tab, the user should disappear from the list entirely.
 
+## Usage With LiveView
+
+Whilst Phoenix does ship with a JavaScript API for dealing with presence, it is also possible to extend the `HelloWeb.Presence` module to support LiveView.
+
+One think to keep in mind when dealing with LiveView, is that each LiveView is a stateful process, so if we keep the oresebce state in the LiveView, each LiveView process will contain the full list of online users in memory. Instead, we can keep track of the online users within the `Presence` process, and pass separate events to the LiveView, which can use a stream to update the online list.
```

Comment:
```suggestion
One think to keep in mind when dealing with LiveView, is that each LiveView is a stateful process, so if we keep the presence state in the LiveView, each LiveView process will contain the full list of online users in memory. Instead, we can keep track of the online users within the `Presence` process, and pass separate events to the LiveView, which can use a stream to update the online list.
```

Reply:
Also, "one thing" instead of "one think"!